store/CHANGELOG.md

```diff
@@ -23,11 +23,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
-> **Disclaimer**: Numbers from v1.0.x to v1.9.x are reserved for the v0.50 line.
-> cosmossdk.io/store compatible with the v0.50 line is tagged from release/v0.50.x
-> Numbers from v1.10.x onwards are reserved for the 0.52+ line.
-> With Cosmos SDK v2 (with store/v2), CometBFT has been pushed to the boundaries, so issues like this
-> are not expected to happen again.
+## v1.1.2 (March 31, 2025)
+
+### Bug Fixes
+
+* [#24090](https://github.com/cosmos/cosmos-sdk/pull/24090) Running the `prune` command disabled async pruning.
```

Comment:
tiny nit but it kinda read like prune command is meant to be async. i think the below is more clear
```suggestion
* [#24090](https://github.com/cosmos/cosmos-sdk/pull/24090) Running the `prune` command now disables async pruning.
```